lacecore/_mesh.py

```diff
@@ -19,6 +19,7 @@ def __init__(self, v, f, copy_v=False, copy_f=False):
         v.setflags(write=False)
         self.f = f
         self.v = v
+        self.segm = segm
```

Comment:
I suggest we call this `face_groups` for consistency with the OBJ language and because it's also possible to have groups of vertices in OBJ, so this makes it clearer what's in there. Could you integrate this with the GroupMap? I'd suggest passing an already-constructed instance of GroupMap to the Mesh constructor. Alternatively you could back out this part and defer it to a separate PR.

Reply:
Done